<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerForm</class>
 <widget class="QWidget" name="registerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>1170</width>
     <height>670</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{ 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  #febf94,stop:0.33 #d6b3b6,stop:0.67 #a4a8d6, stop:1 #519ef6);
	border-radius: 20px;
}
</string>
   </property>
   <widget class="QWidget" name="shell" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>35</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	width: 640px;
	height: 320px;</string>
    </property>
    <widget class="QWidget" name="boxleft" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>145</y>
       <width>400</width>
       <height>310</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">            background-color: #fff;
            height: 280px;
            top: 20px;
            position: relative;
            width: 50%;</string>
     </property>
     <widget class="QLabel" name="LoginLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">	color: #474a59;
    font: 900 50px '';
    margin: 50px 40px 40px;
	font:&quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:30pt; font-weight:600;&quot;&gt;Sign Up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Describe">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>171</y>
        <width>411</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
	line-height:20px;
    color: #999;
    font-style: 14px;
    margin: 40px;
	border:none;
</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'等线'; font-size:9pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:3px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:400;&quot;&gt;Qt治好了我的精神内耗&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="boxright" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>400</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#boxright{
	background-color: #474a59;
	border-radius: 15px;
    color: #f1f1f2;
    width: 50%;
	border-top: 1px solid QLinearGradient(y0:0, y1:1,stop: 0 #DADADA, stop: 1 transparent);
	border-left: 1px solid QLinearGradient(x0:0, x1:1,stop: 0 #DADADA, stop: 1 transparent);
	border-bottom: 1px solid QLinearGradient(y0:0, y1:1,stop: 0 transparent, stop: 1  #DADADA);
	border-right: 1px solid QLinearGradient(x0:0, x1:1,stop:  0 transparent, stop: 1 #DADADA);
}</string>
     </property>
     <widget class="QLabel" name="usersNameLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-10</y>
        <width>370</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#usersNameLabel{
	color: #c2c2c5;
    font-size: 18px;
	font:&quot;Arial Black&quot;;
    margin-top: 20px;
    margin-bottom: 5px;
    position: relative;
}</string>
      </property>
      <property name="text">
       <string>Your Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>370</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#PasswordLabel{
	color: #c2c2c5;
    font-size: 18px;
    margin-top: 20px;
	font:&quot;Arial Black&quot;;
    margin-bottom: 5px;
    position: relative;
}</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usersName">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>320</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#usersName
{
	background: transparent;
    border: 0;
    color: #f2f2f2;
    font-style: 20px;
    height: 30px;
    line-height: 30px;
    width: 100%;
    outline: none !important;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入账号</string>
      </property>
     </widget>
     <widget class="QLabel" name="underLine">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>320</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#underLine
{
    background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  #e38283 ,stop:0.33 #c19f9e, stop:0.67 #91b8bb,stop:1 #1bced8);

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="underLine_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>330</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#underLine_2
{
    background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  #e38283 ,stop:0.33 #c19f9e, stop:0.67 #91b8bb,stop:1 #1bced8);

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>320</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#Password
{
	background: transparent;
    border: 0;
    color: #f2f2f2;
    font-style: 20px;
    height: 30px;
    line-height: 30px;
    width: 100%;
    outline: none !important;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ConfirmBtn">
      <property name="geometry">
       <rect>
        <x>219</x>
        <y>520</y>
        <width>141</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#ConfirmBtn{
			color: #c2c2c5;
            background-color: rgba(255,255,255,.1);
            border-radius: 20px;
            float: right;
        }
#ConfirmBtn:hover{
            color: #474a59;
            background-color: #fff;
}</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ReturnBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>520</y>
        <width>141</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#ReturnBtn{
			color: #c2c2c5;
            border-radius: 20px;
            background-color: rgba(255,255,255,.1);
            float: right;
        }
#ReturnBtn:hover{
            color: #474a59;
            background-color: #fff;
}</string>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="underLine_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>330</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#underLine_3
{
    background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  #e38283 ,stop:0.33 #c19f9e, stop:0.67 #91b8bb,stop:1 #1bced8);

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="PasswordConfirmed">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>320</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#PasswordConfirmed
{
	background: transparent;
    border: 0;
    color: #f2f2f2;
    font-style: 20px;
    height: 30px;
    line-height: 30px;
    width: 100%;
    outline: none !important;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请再次输入密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordConfirmedLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>370</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#PasswordConfirmedLabel{
	color: #c2c2c5;
    font-size: 18px;
    margin-top: 20px;
	font:&quot;Arial Black&quot;;
    margin-bottom: 5px;
    position: relative;
}</string>
      </property>
      <property name="text">
       <string>Confirm Your Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordStrong">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>40</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
color:#c9c9c9;</string>
      </property>
      <property name="text">
       <string>未知</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordStrongLable">
      <property name="geometry">
       <rect>
        <x>249</x>
        <y>140</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线 Light</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#c9c9c9;</string>
      </property>
      <property name="text">
       <string>密码强度：</string>
      </property>
     </widget>
     <widget class="Verification" name="verification_code">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>430</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="verification">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>221</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
border:1px solid #c9c9c9;
border-radius:5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>241</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label{
	color: #c2c2c5;
    font-size: 18px;
    margin-top: 20px;
	font:&quot;Arial Black&quot;;
    margin-bottom: 5px;
    position: relative;
}</string>
      </property>
      <property name="text">
       <string>Verification</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Verification</class>
   <extends>QPushButton</extends>
   <header location="global">verification.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
